exonum/src/runtime/execution_context.rs

```diff
@@ -236,6 +255,11 @@ pub trait ExecutionContextUnstable {
     /// Invokes the interface method of the instance with the specified ID.
     ///
     /// See explanation about [`fallthrough_auth`](struct.ExecutionContext.html#child_context).
+    ///
+    /// # Panics
+    ///
+    /// If this method returns an error, do not access the blockchain data through this context
```

Comment:
I'd add a note that if child call errored, user is expected to propagate the error without any further processing.